src/TodoList.jsx

```diff
@@ -17,7 +19,7 @@ function TodoList() {
   return (
     <ul>
       {toDoList.map(function (item) {
-        return <li key={item.id}>{item.title}</li>;
+        return <TodoListItem id={item.id} todo={item.title} />;
```

Comment:
The `todo` prop must reveive the object.

From the assignment:
> Update the todo object reference to come from props

This is how:
```suggestion
        return <TodoListItem id={item.id} todo={item} />;
```

You also need to change the `TodoListItem` component to support that.